content/en/synthetics/guide/browser-tests-totp.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Time-based one time passwords (TOTPs) for multi-factor authentication (MFA)
+kind: guide
+further_reading:
+   - link: 'https://www.datadoghq.com/blog/mfa-synthetic-testing-datadog/'
+     tag: 'Blog'
+     text: 'Introducing multi-factor authentication in Datadog Synthetic tests'
+   - link: 'synthetics/settings/?tab=specifyvalue#global-variables'
+     tag: 'Documentation'
+     text: 'Learn more about global variables'
+   - link: 'synthetics/browser_tests/actions/'
+     tag: 'Documentation'
+     text: 'Learn more about browser test steps'
+ 
+---
+
```

Comment:
It feels like we're missing some kind of intro about MFA/TOTP here.

Reply:
Austin was suggesting to stick to technical aspects as it is a guide and giving the opportunity to read the blog post for more context.